src/useClickAway.ts

```diff
@@ -3,9 +3,9 @@ import { off, on } from './util';
 
 const defaultEvents = ['mousedown', 'touchstart'];
 
-const useClickAway = (
+const useClickAway = <T extends Event>(
```

Comment:
Would default help here?

```suggestion
const useClickAway = <T extends Event = Event>(
```